pom.xml

```
@@ -44,7 +44,7 @@ THE SOFTWARE.
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/design-library-plugin</gitHubRepo>
-        <jenkins.version>2.350</jenkins.version>
+        <jenkins.version>2.365</jenkins.version>
```

```suggestion
        <jenkins.version>2.364</jenkins.version>
```

nit: That's the version the rename was shipped initially